packages/assets-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.0]
+### Added
+- **BREAKING**: Add required options `getSelectedAddress` and `getMultiAccountBalancesEnabled` to AccountTrackerController constructor and make use of them when refreshing account balances ([#1146](https://github.com/MetaMask/core/pull/1146))
+  - Previously, the controller would refresh all account balances, but these options can be used to only refresh the currently selected account
+- **BREAKING:** Add logic to support validating and adding ERC721 and ERC1155 tokens to NFTController state via `wallet_watchAsset` API. ([#1173](https://github.com/MetaMask/core/pull/1173), [#1406](https://github.com/MetaMask/core/pull/1406))
+  - The `NFTController` now has a new `watchNFT` method that can be called to send a message to the `ApprovalController` and prompt the user to add an NFT to their wallet state.
+  - The `NFTController` now requires an instance of a ControllerMessenger to be passed to its constructor. This is messenger is used to pass the `watchNFT` message to the `ApprovalController`.
+
+### Changed
+- Add dependency on `@ethersproject/address` ([#1173](https://github.com/MetaMask/core.git/pull/1173))
```

Comment:
Does this need to be included in the changelog?

Reply:
cc @Gudahtt

Reply:
Historically we have been listing added/changed dependencies in the changelog (as long as they are runtime or peer dependencies, not development dependencies). For instance, see https://github.com/MetaMask/core/blob/main/packages/gas-fee-controller/CHANGELOG.md#600 or https://github.com/MetaMask/core/blob/main/packages/approval-controller/CHANGELOG.md#300). This advises consumers of changes to the dependency tree that would occur with an upgrade to this version. For us it especially lets us know when we might get new Socket alerts or when there might be new changes to LavaMoat policy files.